compose/mpp/demo/src/wasmJsMain/resources/index.html

```diff
@@ -35,6 +37,6 @@
 
 <body>
     <h1>compose wasm web demo</h1>
-    <canvas id="canvas1" width="800" height="600"></canvas>
+    <canvas id="canvas1"></canvas>
```

Comment:
what's the purpose of this change? I mean, the PR is about touch events - how does the viewport area affects this issue?

Reply:
I guess the sizes here were redundant and irrelevant, since CanvasBasedWindow resizes the canvas according to the window size by default.

Reply:
@jershell but this is irrelevant to the touch events as well - I suggest just push us to push both of the html/layout related changes right after we'll push this PR dedicated to changes in touch as a separate commit.